src/miner.cpp

```diff
@@ -251,9 +251,10 @@ CBlockTemplate* Mining::CreateNewBlock(const CChainParams& chainparams) const
             if (!IsFinalTx(tx, nHeight, nLockTimeCutoff))
                 continue;
 
+            const uint64_t maxSigOps = Policy::blockSigOpAcceptLimit(nBlockSize + nTxSize);
```

Comment:
Thank you for catching this! I have modified the code to subtract the coinbase reservation size when calculating the sigop limit in `CreateNewBlock`.